build-support/bin/ci.py

```diff
@@ -322,32 +349,66 @@ def run_python_tests_v1() -> None:
       green("Contrib unit tests passed.")
 
 
-def run_python_tests_v2() -> None:
-  known_v2_failures_file = "build-support/unit_test_v2_blacklist.txt"
-  with open(known_v2_failures_file, "r") as f:
-    blacklisted_targets = {line.strip() for line in f.readlines()}
-  with travis_section("PythonTestsV1", "Running Python unit tests with V2 test runner"):
+def run_python_tests_v2(*, remote_execution_enabled: bool) -> None:
```

Comment:
I think that shrinking this code is pretty important. I've opened https://github.com/pantsbuild/pants/issues/8087 because dogfooding our own facilities during rollouts like this is useful.